app/models/user/academics/diploma.rb

```diff
@@ -0,0 +1,33 @@
+module User
+  module Academics
+    # Provides diploma data related to students
+    class Diploma
+      attr_accessor :user
+
+      def initialize(user)
+        self.user = user
+      end
+
+      def sso_url
+        data[:ucSrCediplomaUrl]
+      end
+
+      def as_json(options = {})
+        {
+          sso_url: sso_url
+        }
+      end
+
+      private
+
+      def data
+        return {} unless user.is_student?
+        @enrollments_data = CampusSolutions::CeDiplomaSso
+        .new({ user_id: user.uid }).get[:feed][:root][:ucSrCediploma]
```

Comment:
I think this formatting is unusual. Also, memoize so repeatedly calling `sso_url` doesn't issue multiple requests. I would format more like this.

```
@enrollments_data ||= CampusSolutions::CeDiplomaSso.new({
  user_id: user.uid
}).get[:feed][:root][:ucSrCediploma]
```